lib/pynput/keyboard/_darwin.py

```diff
@@ -163,12 +189,30 @@ class Key(enum.Enum):
     tab = KeyCode.from_vk(0x30)
     up = KeyCode.from_vk(0x7E)
 
-    media_play_pause = KeyCode.from_vk(NX_KEYTYPE_PLAY, _is_media=True)
-    media_volume_mute = KeyCode.from_vk(NX_KEYTYPE_MUTE, _is_media=True)
-    media_volume_down = KeyCode.from_vk(NX_KEYTYPE_SOUND_DOWN, _is_media=True)
-    media_volume_up = KeyCode.from_vk(NX_KEYTYPE_SOUND_UP, _is_media=True)
-    media_previous = KeyCode.from_vk(NX_KEYTYPE_PREVIOUS, _is_media=True)
-    media_next = KeyCode.from_vk(NX_KEYTYPE_NEXT, _is_media=True)
+    media_play_pause = KeyCode.from_vk(tKeyNameToCode['NX_KEYTYPE_PLAY'], _is_media=True)
```

Comment:
When merging, I will prune this list to contain only the keys defined in the base. If these keys are later added to base, your PR will act as reference.

Reply:
Alright, what would require for new keys to be added to base? I'm quite interested in using NX_KEYTYPE_BRIGHTNESS_UP and NX_KEYTYPE_BRIGHTNESS_DOWN.